docs/antora-build.mk

```diff
@@ -1,4 +1,5 @@
 docs_out_dir := ./.public
+docs_moduleroot_dir := docs/modules/ROOT
```

Comment:
Why this change? doesn't seem to be used.

Reply:
It is used in [Makefile](https://github.com/vshn/provider-exoscale/blob/master/Makefile#L70). Unfortunately rewriting the commit history is not always perfect.